bluesky/log.py

```diff
@@ -125,22 +124,70 @@ def format(self, record):
         return formatted.replace("\n", "\n    ")
 
 
-plain_log_format = "[%(levelname)1.1s %(asctime)s.%(msecs)03d %(module)s:%(lineno)d] %(message)s"
+plain_log_format = "[%(levelname)1.1s %(asctime)s.%(msecs)03d %(module)15s:%(lineno)5d] %(message)s"
 color_log_format = ("%(color)s[%(levelname)1.1s %(asctime)s.%(msecs)03d "
-                    "%(module)s:%(lineno)d]%(end_color)s %(message)s")
+                    "%(module)15s:%(lineno)5d]%(end_color)s %(message)s")
+
+
+def color_logs(color):
```

Comment:
This is no need to add this function. In the implementation that this was copied from, the function is deprecated.

Reply:
Ok